Integrations/DomainTools_Iris/iris.py

```diff
@@ -0,0 +1,393 @@
+    threat_profile_data = get_threat_component(domain_risk['components'], 'threat_profile')
+    if threat_profile_data:
+        threat_profile_threats = ', '.join(threat_profile_data.get('threats', []))
+        threat_profile_evidence = ', '.join(threat_profile_data.get('evidence', []))
+    else:
+        threat_profile_threats = None
+        threat_profile_evidence = None
+
+    human_readable_data = {
+        "Overall Risk Score": overall_risk_score,
+        "Threat Profile Threats": threat_profile_threats,
+        "Threat Profile Evidence": threat_profile_evidence
+    }
+
+    outputs = {}  # type: Dict[Any, Any]
+
+    headers = ['Overall Risk Score', 'Threat Profile Threats', 'Threat Profile Evidence']
+    human_readable = tableToMarkdown('DomainTools Threat Profile for {}.'.format(domain_result['domain']),
+                                     human_readable_data,
+                                     headers=headers)
+    return human_readable, outputs
+
+
+def format_domain_pivot(response, search_type, search_value):
+    domain_list = [x['domain'] for x in response['results']]
+    human_readable_data = domain_list
+    domain_context_list = []
+
+    for domain_result in response['results']:
+        domain_context = create_context_output(domain_result)
+        domain_context_list.append(domain_context)
+    outputs = {"PivotedDomains(val.Name == obj.Name)": domain_context_list}
+    convert_empty_to_null(outputs)
+    headers = ['Domains']
+    human_readable = tableToMarkdown('Domains for {}: {}.'.format(search_type, search_value),
+                                     human_readable_data,
+                                     headers=headers)
+    return human_readable, outputs
+
+
+''' COMMANDS '''
+
+
+def domain_profile():
+    res = http_request('GET', '/v1/iris-investigate/', {'domain': demisto.args().get('domain')})
+    if res.get('results_count'):
+        human_readable, outputs = format_domain_profile(res)
+        return_outputs(human_readable, outputs, res)
+    else:
+        demisto.results('No results found.')
+
+
+def domain_analytics():
+    res = http_request('GET', '/v1/iris-investigate/', {'domain': demisto.args().get('domain')})
+    if res.get('results_count'):
+        human_readable, outputs = format_domain_analytics(res)
+        return_outputs(human_readable, outputs, res)
+    else:
+        demisto.results('No results found.')
+
+
+def threat_profile():
+    res = http_request('GET', '/v1/iris-investigate/', {'domain': demisto.args().get('domain')})
+    if res.get('results_count'):
+        human_readable, outputs = format_threat_profile(res)
+        return_outputs(human_readable, outputs, res)
+    else:
+        demisto.results('No results found.')
+
+
+def domain_pivot():
+    res = None  # type: ignore
+    search_type = ""
+    search_value = ""
+    if demisto.args().get('ip'):
+        res = http_request('GET', '/v1/iris-investigate/', {'ip': demisto.args().get('ip')})
```

Comment:
Let's just initialise the parameters here and call `http_request` after the conditions block